### PR TITLE
Add new repo openshift/assisted-test-infra config and job

### PR DESCRIPTION
**Pull request by YuviGold**
New config for [assisted-test-infra](https://github.com/openshift/assisted-test-infra) repository.
For now just a sanity check, will be updated later to build a cluster on top of packet and run openshift tests

### CODE REVIEW

File: ci-operator/config/openshift/assisted-test-infra/OWNERS [hunk 1]
```diff
@@ -0,0 +1,5 @@
+approvers:
+- tsorya
```
Comment (romfreiman) [inline ci-operator/config/openshift/assisted-test-infra/OWNERS:2]: eran

Comment (romfreiman) [reply ci-operator/config/openshift/assisted-test-infra/OWNERS:2]: shouldnt it be part of the assisted-test-infra repo?

File: ci-operator/jobs/openshift/assisted-test-infra/OWNERS [hunk 2]
```diff
@@ -0,0 +1,5 @@
+approvers:
```
Comment (romfreiman) [inline ci-operator/jobs/openshift/assisted-test-infra/OWNERS:1]: eran

I think there is also a section of reviewer?

File: ci-operator/config/openshift/assisted-test-infra/OWNERS (hunk 1 above)
Comment (YuviGold) [reply ci-operator/config/openshift/assisted-test-infra/OWNERS:2]: > eran

Isn't a member of OpenShift organization.

> shouldnt it be part of the assisted-test-infra repo?

Maybe in the future. Right now Prow needs an OWNERS file for each directory.

**Pull request merged**